src/README.md

```diff
@@ -158,6 +158,10 @@ npm install prosemirror-utils
 
 @convertArrayOfRowsToTableNode
 
+### Deprecated methods
+
+@cloneRowAt
```

Comment:
A few thoughts on this:
* normally any breaking change goes through an RFC, where people comment of the change, since its a public API it should consider all use cases, not only ours. 
* a nicer way of deprecating a method would be to have a console warning, mentioning when it will be deprecated and what is the transition path.

Reply:
I totally agree with those points, but we don't have any process for that, and looks like we need to create one CONTRIBUTING to describe that process.